src/backend/api/reservations.js

```diff
@@ -0,0 +1,118 @@
+import express from "express";
+const router = express.Router();
+import knex from "../database.js";
+
+// /api/reservations	GET	Returns all reservations
+router.get('/', async (req, res) => {
+    try {
+        const reservations = await knex.select().from('reservation');
+        if (reservations) {
+            res.json(reservations);
+        }else {
```

Comment:
since I do not have the .env file I cannot probably run the project but I assume that you are using array, right? If so, that's a bit weird part of javascript but both empty arrays and objects are actually true.. 🥲 so if you want to send reservations only if there are some reservations I recommend using `if(reservations.length)` instead 👍 but good work. Also please do not forget to send the status code always

Reply:
I will try this ```if(reservations.length)```